packages/design-system/src/index.ts

```diff
@@ -11,6 +11,7 @@ export {
   theme,
   type CSS,
 } from "./stitches.config";
+export { toast, Toaster } from "./components/toast";
```

Comment:
I started exporting specific components but its tedious, would only do that if there is something to hide I a component

```suggestion
export * from "./components/toast";
```

Reply:
Also done!